migrations/20240823213817_add_feedback.up.sql

```diff
@@ -0,0 +1,28 @@
+-- Add up migration script here
+
+CREATE TYPE feedback_type AS ENUM
+    ('Positive', 'Negative', 'Suggestion', 'Note');
+
+CREATE TABLE mod_feedback
+(
+    id SERIAL PRIMARY KEY NOT NULL,
+    mod_version_id INTEGER NOT NULL,
+    reviewer_id INTEGER NOT NULL,
+    feedback TEXT COLLATE pg_catalog."default" NOT NULL,
+    decision BOOLEAN NOT NULL DEFAULT false,
+    type feedback_type NOT NULL,
+    dev bool NOT NULL DEFAULT false,
```

Comment:
this could get a little confusing, considering that "this person is a developer of this mod" is not constant. i'd rather this check be done during the sql query (like the admin field) instead of stored in the table

Reply:
ah alr